synapse/crypto/context_factory.py

```diff
@@ -32,8 +32,11 @@ def __init__(self, config):
     @staticmethod
     def configure_context(context, config):
         try:
-            _ecCurve = _OpenSSLECCurve(_defaultCurveName)
-            _ecCurve.addECKeyToContext(context)
+            # This was removed in https://github.com/twisted/twisted/pull/928
+            # _ecCurve = _OpenSSLECCurve()
```

Comment:
not much point in leaving it here as a comment, then.

Reply:
Left here mainly as a tombstone since it's a drop in replacement mostly, but to be fair git serves that purpose.